fog/view/server/src/lib.rs

```diff
@@ -12,3 +12,5 @@ pub mod server;
 mod block_tracker;
 mod counters;
 mod db_fetcher;
+mod fog_view_router_request_handler;
+mod fog_view_shard_responses_processor;
```

Comment:
I'd suggest these don't need to be this verbose---like, the full path is: `mc_fog_view_server::fog_view_router_request_hander`. At the very least, I'd nuke the `fog_view` prefixes.

Reply:
Nuked the prefixes.